src/Responder/FormattedResponder.php

```diff
@@ -14,11 +14,16 @@
 use Equip\Resolver\ResolverTrait;
 use Relay\ResolverInterface;
 
-class FormattedResponder extends Dictionary implements ResponderInterface
+class FormattedResponder extends SortedDictionary implements ResponderInterface
 {
     use ResolverTrait;
 
     /**
+     * @var callable
+     */
+    protected $sorter = 'arsort';
```

Comment:
Maybe I'm a bit confused on this, but won't this sort regardless of order added?